package/Dockerfile

```diff
@@ -31,14 +31,15 @@ RUN cd /build-xen && \
 
 COPY . /build
 RUN cd /build/libvmi && \
-    cmake -DENABLE_KVM=OFF -DENABLE_FILE=OFF -DENABLE_BAREFLANK=OFF -DCMAKE_INSTALL_PREFIX=/usr . && \
+    cmake -DENABLE_KVM=OFF -DENABLE_FILE=OFF -DENABLE_BAREFLANK=OFF -DCMAKE_INSTALL_PREFIX=/usr/local . && \
```

Comment:
`/usr/local` is for building from sources, for a normal package I would suggest to stick to `/usr` prefix. It has the advantage that somebody could install from sources over the existing `.deb` package without overriding it.

Also, a problem with `/usr/local` is that you have to set `LD_LIBRARY_PATH` manually and sometimes this is a bit annoying. My code was intended to run "out of the box" as DEBs a generally suposed to. Even if it's experimental, it comes from package so would be good to stick to `/usr` convention.